benchmarks/_sources.py

```diff
@@ -285,9 +366,24 @@ def csv_parse_options(self):
     @property
     def csv_read_options(self):
         if self.store["header"] is None:
-            return pyarrow.csv.ReadOptions(autogenerate_column_names=True)
+            autogenerate_column_names = self.store["header"] is None
```

Comment:
this is always True, no?

Reply:
No, the nyctaxi csvs have `"header": 0` for whatever reason. Maybe I should just go back and turn all the metadata into booleans to make it all clearer

Reply:
haha nice love it